### PR TITLE
Feature/ci 149 fetch toyboo creation

### PR DESCRIPTION
**Pull request by JakubJankowski**
Jira: https://nomtek.atlassian.net/browse/CI-149
API Ref: https://stateoftheart.creatubbles.com/api/#toyboo-creation-details

### CODE REVIEW

File: CreatubblesAPIClient/Sources/Model/ToybooCreation.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//  Created by Nomtek on 02.11.2016.
+//  Copyright © 2016 Nomtek. All rights reserved.
+//
+
+import UIKit
+
+@objc
+public class ToybooCreation: NSObject
+{
+    public let identifier: String
```
Comment (mmiedlarz) [inline CreatubblesAPIClient/Sources/Model/ToybooCreation.swift:14]: are you sure, that `uzpbUrl` and `contentUrl` are not optional? (They're not documented anywhere).

File: CreatubblesAPIClientTests/Spec/Creation/FetchToybooCreationResponseHandlerSpec.swift [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+class FetchToybooCreationResponseHandlerSpec: QuickSpec
+{
+    override func spec()
+    {
+        describe("Fetch Toyboo Creation response handler")
+        {
+            it("Should return a correct creation after login")
+            {
+                let request = FetchToybooCreationRequest(creationId: "kniSnnvO")
```
Comment (mmiedlarz) [inline CreatubblesAPIClientTests/Spec/Creation/FetchToybooCreationResponseHandlerSpec.swift]: I would creationId to `TestConfiguration`

File: CreatubblesAPIClientTests/Spec/Creation/FetchToybooCreationResponseHandlerSpec.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                            sender.logout()
+                            done()
+                        })
+                    }
+                }
+            }
+            
+            it("Should return an error when not logged in")
+            {
+                let request = FetchToybooCreationRequest(creationId: "kniSnnvO")
```
Comment (mmiedlarz) [inline CreatubblesAPIClientTests/Spec/Creation/FetchToybooCreationResponseHandlerSpec.swift]:
``` swift
guard let creationId = TestConfiguration.toyboCreationIdentifier
else { return }
let request = FetchToybooCreationRequest(creationId: creationId)
...
```

**Comment by mmiedlarz** [comment]
Code 👍 
Func N/A

File: CreatubblesAPIClient/Sources/Model/ToybooCreation.swift (hunk 1 above)
Comment (JakubJankowski) [reply CreatubblesAPIClient/Sources/Model/ToybooCreation.swift:14]: Yeah, I asked Michael and he said they'll always be present.

**Pull request merged**